### DISCUSSION TITLE
Geofence name with supercharger not included in some dashboards

### DISCUSSION THREAD

**Discussion by vincep5** [jheredianet/Teslamate-CustomGrafanaDashboards#32] [General]
Some of the queries should be updated so that if I make a Geofence for Tesla Superchargers, they get included.
Some of my recent charging sessions didnt have the address name with the text "Supercharger" so they seem to be excluded from several dashboards and its queries.  Example of one is the Charging Costs Stats

**Comment by jheredianet** [comment]
Hi @vincep5 I've just released a fix on that, could you check it please?

https://github.com/jheredianet/Teslamate-CustomGrafanaDashboards/releases/tag/v2.8.4

**Reply by vincep5** [reply]
I tried it out, and it looks great.

**Reply by jheredianet** [reply]
Thanks for your feedback. 
Cheers!!!
